src/main/java/org/cryptomator/linux/tray/AppindicatorTrayMenuController.java

```diff
@@ -46,9 +46,11 @@ public void showTrayIcon(Consumer<TrayIconLoader> iconLoader, Runnable runnable,
 
 	private void showTrayIconWithSVG(String s) {
 		try (var arena = Arena.openConfined()) {
-			indicator = app_indicator_new(arena.allocateUtf8String(APP_INDICATOR_ID),
+			indicator = app_indicator_new_with_path(arena.allocateUtf8String(APP_INDICATOR_ID),
 					arena.allocateUtf8String(s),
-					APP_INDICATOR_CATEGORY_APPLICATION_STATUS());
+					APP_INDICATOR_CATEGORY_APPLICATION_STATUS(),
+					// find tray icons theme in mounted AppImage
+					arena.allocateUtf8String(System.getenv("APPDIR") + "/usr/share/icons/hicolor/symbolic/apps"));
```

Comment:
`System.getenv(String s)` returns `null`, if the `s` is not defined in the environment. Concatenating null with a string x leads to "nullx". I don't think this is the intended behaviour.

Also, how are double slashes (e.g., `path/to//file`) file handled?